LICs.c

```diff
@@ -108,7 +108,7 @@ double triangle_area (double a, double b, double c) {
 
 //LIC 3
 boolean check_lic_3 () {
-    for(int i = 0; i < 97; i++) {
+    for(int i = 0; i < NUMPOINTS; i++) {
```

Comment:
I think this should be i < NUMPOINTS - 2

Reply:
Oh yeah true